go/test/endtoend/vtgate/gen4/gen4_test.go

```diff
@@ -187,26 +187,6 @@ func TestSubQueriesOnOuterJoinOnCondition(t *testing.T) {
 	}
 }
 
-func TestPlannerWarning(t *testing.T) {
```

Comment:
Unfortunately, `straight_join` no longer produces a warning, and we don't have reliable ways of getting warnings from the planner without turning of schema tracking